### ISSUE TITLE
VisExport pdf doesn't produce a vectorial image

### ISSUE THREAD

**Issue by skanskan** [datastorm-open/visNetwork#209]
I've tried 
%>%  visExport( type="pdf",name="name", label = "button")}
and unfortunatelly the pdf generated doesn't contain a vectorial image of my network but a bad quality raster image.

How can I change this behaviour.

**Comment by deisygysi** [comment]
I'm also having the same problem. The png is also in bad quality :(

**Comment by skanskan** [comment]
This is because it doesn't export the object as a vectorial image, it just takes a screenshot.

**Comment by deisygysi** [comment]
Yes, I noticed that. Isn't there any (quick) way to export is a high-quality image? I've been transforming things to igraph and exporting as pdf. But it is Ok when I have small graphs, but I work with high dimensional data, meaning that, small graphs are exceptions...

**Comment by zx8754** [comment]
Related issue https://github.com/datastorm-open/visNetwork/issues/204
